### ISSUE TITLE
Accessibility Feature in Browser Extension missing

### ISSUE THREAD

**Issue by Trimilur** [bitwarden/clients#20030]
### Steps To Reproduce

The first login of the browser add-in does not include an aria-label for the user-email field. This is required by many accessibility tools. 

### Expected Result

I herewith request to add: required aria-label ="user e-mail" or similar to the browser add in code.

The aria-label for all relevant fields should be available.

### Actual Result

label not available, accessibility not guaranteed.

### Operating System

Windows

### Web Browser

Chrome

### Environment Versions

2026.2.0

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by djsmith85** [comment]
Hi @Trimilur and thank you for the report.

In case this is reproducible, the login page can be found at https://github.com/bitwarden/clients/blob/main/libs/auth/src/angular/login/login.component.html#L17

The form field and label should be accessible, as they are part of the [Bitwarden component library](https://components.bitwarden.com/?path=/docs/component-library-form-field--docs) and used through-out the applications. Maybe this only occurs on certain browser. Does the issue only occur in the popup or also when popped out?

Marking this as good-first-issue in case anyone from the community wants to have a look into this.

Kind regards,
Daniel

**Comment by vleague2** [comment]
Hi @Trimilur, I have tested this with VoiceOver on Mac and the "email address" field on the browser login screen is reading out its label as expected. It does not have an `aria-label` because it is using a properly-associated semantic label. Can you provide more details about what issue you are experiencing? Thanks!

**Comment by Trimilur** [comment]
Hi, thanks for the investigation. Actually it was a wrong information. After a second test everything worked fine. Regards
